CHANGELOG.md

```diff
@@ -14,31 +16,53 @@ might need to be adjusted and either explicitly name the positional
 argument or resolve the type ambiguity by casting to the intended type.
 
 **Features**
+* PG15 support added
+* MultiNode High Avalibility improvements
+* #4335 Show warnings when not following best practices
```

Comment:
```suggestion
```
This is the issue but we don't link issues here but PRs and the PR implementing this is 3 lines below this one.

Reply:
Removed